### PR TITLE
Add basic info for Isle of Armor DLC new moves

### PR DESCRIPTION
**Pull request by ercdndrs**
Hi and thank you for your work creating and maintaining this wonderful resource. I was hoping to use it for a project but noticed that the new moves introduced with the Isle of Armor DLC are missing. I manually entered the information found on Serebii (https://www.serebii.net/swordshield/expansionpass/theisleofarmor.shtml) into moves.csv and also updated the move names file. The other change was adding a new move target, "All Allies", which I believe the new move Coaching is the first to use. This is my first time contributing to this project to please let me know if I have neglected to update something important. Thanks!

### CODE REVIEW

File: pokedex/data/csv/move_target_prose.csv [hunk 1]
```diff
@@ -27,3 +27,4 @@ move_target_id,local_language_id,name,description
 13,9,User and allies,The user and its allies.
 14,6,Jedes Pokémon,Jedes Pokémon auf dem Feld.
 14,9,All Pokémon,Every Pokémon on the field.
+15,9,All Allies,All of the user's allies
```
Comment (Naramsim) [inline pokedex/data/csv/move_target_prose.csv:30]: I'd go here with _All allies_, thus lowercasing _Allies_

**Comment by Naramsim** [comment]
Cool! Thanks so much! I did a quick review and everything seems fine! (apart from an upper to lowercase)

https://bulbapedia.bulbagarden.net/wiki/List_of_moves

I noticed that there are 8 other moves after Surging Strikes. Would you be so kind to also add those :) If you cannot, I'll add them, so no worries.

File: pokedex/data/csv/move_target_prose.csv (hunk 1 above)
Comment (ercdndrs) [reply pokedex/data/csv/move_target_prose.csv]: Fixed and also added a period at the end of the last field.

**Comment by ercdndrs** [comment]
> Cool! Thanks so much! I did a quick review and everything seems fine! (apart from an upper to lowercase)
> 
> https://bulbapedia.bulbagarden.net/wiki/List_of_moves
> 
> I noticed that there are 8 other moves after Surging Strikes. Would you be so kind to also add those :) If you cannot, I'll add them, so no worries.

Done! I had missed the new Crown Tundra moves

**Comment by ercdndrs** [comment]
Just a note that I also added translations for abilities introduced in Gen 8 (at least the translations available on Bulbapedia).

**Comment by Naramsim** [comment]
Yeah, I saw it! Thanks a lot :) I'm merging these and transfer over at PokeAPI when I'll have some time.

**Review by Naramsim — APPROVED** [review]

**Comment by Naramsim** [comment]
I added you to the contributors' page, @ercdndrs. https://pokeapi.co/about

Would you like me to put your real name there?

**Comment by ercdndrs** [comment]
> I added you to the contributors' page, @ercdndrs. https://pokeapi.co/about
> 
> Would you like me to put your real name there?

I don't mind either way.

**Pull request merged**